models/demos/mamba/tests/test_mamba_block.py

```diff
@@ -82,3 +86,72 @@ def test_mamba_block_inference(
     if not does_pass:
         logger.warning("Mamba block output failed")
         assert does_pass, f"PCC value is lower than {pcc}"
+
+
+class PytorchMambaBlockPrefill(torch.nn.Module):
```

Comment:
We should be consistent with the naming in the other test. Maybe `PytorchPrefillMambaBlock` or change the SSM test to `PytorchMambaSSMPrefill`.

Reply:
I like the `PytorchPrefillMambaBlock`.